docker-compose.yaml

```diff
@@ -1,5 +1,13 @@
 version: '3'
 services:
+  pgadmin:
+    image: dpage/pgadmin4
+    environment:
+      PGADMIN_DEFAULT_EMAIL: checkins@objectcomputing.com
+      PGADMIN_DEFAULT_PASSWORD: admin
+    ports:
+      - "80:80"
+    container_name: pgadmin
```

Comment:
In the future, it might be worth splitting sonarqube and pgadmin into a separate docker-compose file. Maybe a `docker-compose-tools.yaml` or some such...